.gitignore

```diff
@@ -18,3 +18,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.yarnclean
```

Comment:
Aparently the react-boilerplate project also includes these files. Let's not ignore them then even though they are annoying.

Reply:
removed